### PR TITLE
optimism-bedrock: rollup changes

### PR DESCRIPTION
**Pull request by tynes**
PR to easily view the diff from upstream geth for review purposes

Do not merge

### CODE REVIEW

File: core/beacon/gen_blockparams.go [hunk 1]
```diff
@@ -15,14 +15,23 @@ var _ = (*payloadAttributesMarshaling)(nil)
 // MarshalJSON marshals as JSON.
 func (p PayloadAttributesV1) MarshalJSON() ([]byte, error) {
 	type PayloadAttributesV1 struct {
-		Timestamp             hexutil.Uint64 `json:"timestamp"     gencodec:"required"`
-		Random                common.Hash    `json:"random"        gencodec:"required"`
-		SuggestedFeeRecipient common.Address `json:"suggestedFeeRecipient"  gencodec:"required"`
+		Timestamp             hexutil.Uint64  `json:"timestamp"     gencodec:"required"`
+		Random                common.Hash     `json:"random"        gencodec:"required"`
```
Comment (tynes) [inline core/beacon/gen_blockparams.go:19]: As of kiln v2, the `Random` field is now `prevRandao` in the JSON
https://github.com/ethereum/go-ethereum/commit/4f4622bc8b104fc87c5ed64ed86895fe118e1ea7

Comment (protolambda) [reply core/beacon/gen_blockparams.go:19]: That change is not yet included in this version of geth we are building on.

Comment (protolambda) [reply core/beacon/gen_blockparams.go:19]: We should probably just rebase the commit on the latest geth to get all bug-fixes, and then make these minor changes in an update to the rollup-node.

Comment (protolambda) [reply core/beacon/gen_blockparams.go]: Fixed in #8, please continue rollup review comments there.

**Comment by protolambda** [comment]
Superseded by #8, now rebased on the latest geth `master` branch.

**Pull request closed without merging**